package.json

```diff
@@ -30,6 +33,7 @@
     "@angular/cli": "1.6.5",
     "@angular/compiler-cli": "^5.2.0",
     "@angular/language-service": "^5.2.0",
+    "@compodoc/compodoc": "^1.1.2",
```

Comment:
Please add dependencies alphabetically.

Reply:
@RishabhJain2018 I didn't intend to add the dependency alphabetically but it looks like it is already added at the right spot 😛 Isn't it ?

Reply:
ohh..yes! I noticed only `language-service` and `compodoc`. That's why I added the comment. But yes it is alphabetical. 😄